Src/IronPythonTest/EngineTest.cs

```diff
@@ -1079,7 +1077,7 @@ class ns(object):
     def __int__(self): return 42
     def __float__(self): return 42.0
     def __str__(self): return 'Python'
-    def __long__(self): return long(42)
+    def __long__(self): raise TypeError
```

Comment:
I'm fine with removing it. I kept it here because it is a test and wanted to make sure that no code path calls `__long__` anymore. Although I grepped for `__long__`, sometimes the reflection code composes a method name from parts, so you never know...